src/Raven.Server/Config/Categories/PatchingConfiguration.cs

```diff
@@ -1,22 +1,11 @@
 using System.ComponentModel;
 using Raven.Server.Config.Attributes;
+using Raven.Client.ServerWide.JavaScript;
 
 namespace Raven.Server.Config.Categories
 {
     [ConfigurationCategory(ConfigurationCategoryType.Patching)]
     public class PatchingConfiguration : ConfigurationCategory
     {
-        /// <summary>
-        /// The maximum number of steps iterations to give a script before timing out.
-        /// Default: 10000
-        /// </summary>
-        [DefaultValue(10_000)]
-        [ConfigurationEntry("Patching.MaxStepsForScript", ConfigurationEntryScope.ServerWideOrPerDatabase)]
-        public int MaxStepsForScript { get; set; }
-
-        [Description("Enables Strict Mode in JavaScript engine. Default: true")]
-        [DefaultValue(true)]
-        [ConfigurationEntry("Patching.StrictMode", ConfigurationEntryScope.ServerWideOrPerDatabase)]
-        public bool StrictMode { get; set; }
```

Comment:
Right, returned them back and implemented backward compatibilty of JS options: using PatchingConfiguration options in case they were used before and set by the user